Move APIs for service pages (#3512)
viamrobotics/docs: docs/appendix/apis/services/navigation.md

---
title: "Navigation service API"
linkTitle: "Navigation"
weight: 50
type: "docs"
tags: ["navigation", "services", "base", "rover"]

> description: "The navigation service API allows you to define waypoints and move your machine along those waypoints while avoiding obstacles."

Author: ```suggestion
description: "Give commands to define waypoints and move your machine along those waypoints while avoiding obstacles."
```
